scripts/deployment/globals_gnosis_mainnet_qs_beta_expert4.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
why no agentMechAddress?

Reply:
Not needed, as the activity checker is based on `livenessRatio` only.

Reply:
done